ic-helper.ps1

```diff
@@ -66,22 +66,35 @@ function Find-GitRepositories {
         [string]$command,
         [switch]$force
     )
-    $subdirs = Get-ChildItem -Path $path -Directory -Recurse
+
+ # Check if the path exists
+    if (-not (Test-Path $path)) {
+        Write-Error "The specified path '$path' does not exist."
+         return
+    }
+
+    try {
+        $subdirs = Get-ChildItem -Path $path -Directory -Recurse
+    }
+    catch {
+        Write-Error "An error occurred while retrieving subdirectories: $($_.Exception.Message)"
+        return
```

Comment:
Always use `exit 1` instead of `return` when you want to stop the script after catching an error